openml-lightgbm/lightgbm-provider/src/test/java/com/feedzai/openml/provider/lightgbm/LightGBMBinaryClassificationModelTrainerTest.java

```diff
@@ -438,13 +438,17 @@ static ArrayList<List<Double>> fitModelAndGetFirstScoresPerClass(
                 maxInstancesToTrain
         );
 
-        final LightGBMBinaryClassificationModel model = fit(
+        try (final LightGBMBinaryClassificationModel model = fit(
                 dataset,
                 modelParams,
                 chunkSizeInstances
-        );
-
-        return getClassScores(dataset, model, maxInstancesToScore);
+        )) {
+            return getClassScores(dataset, model, maxInstancesToScore);
+        } catch (ModelLoadingException e) {
```

Comment:
```suggestion
        } catch (URISyntaxException | IOException | ModelLoadingException e) {
```

and see if we can still add the final to that (`final a|b|c e`).